config/mutable_test.go

```diff
@@ -43,10 +43,19 @@ func TestGlobalCreateBucket(t *testing.T) {
 }
 
 func TestCreateBucket(t *testing.T) {
-	cfg := defaultConfig()
+	nilMapBucketsCfg := defaultConfig()
```

Comment:
Can we make this a separate test function? E.g., `func TestCreateBucketInEmptyNamespace`?